[Bug] Cloudflare Pagesデプロイが wrangler 設定不足で失敗する

## 説明

Cloudflare Pagesのデプロイが `npx wrangler deploy` 実行時に「Missing entry-point to Worker script or to assets directory」エラーで失敗する。プロジェクトに `wrangler.jsonc`（または `wrangler.toml`）設定ファイルが存在せず、wranglerがビルド出力ディレクトリ（`./dist`）を認識できないことが原因。

## 再現手順

1. Cloudflare Pages でビルドがトリガーされる
2. `pnpm install --frozen-lockfile` が成功する
3. `pnpm run build`（`tsc -b && vite build`）が成功し、`dist/` にビルド出力が生成される
4. デプロイコマンド `npx wrangler deploy` が実行される
5. wrangler が設定ファイルを見つけられず、エラーで失敗する

## 期待される動作

`vite build` で生成された `dist/` ディレクトリの静的アセットが Cloudflare Pages に正常にデプロイされる。

## 実際の動作

以下のエラーでデプロイが失敗する:

```
✘ [ERROR] Missing entry-point to Worker script or to assets directory

  If are uploading a directory of assets, you can either:
  - Specify the path to the directory of assets via the command line: (ex: npx wrangler deploy --assets=./dist)
  - Or create a "wrangler.jsonc" file containing:

  {
    "name": "worker-name",
    "compatibility_date": "2026-02-07",
    "main": "src/index.ts"
  }
```

## スクリーンショット

N/A（デプロイログは説明欄に記載）

## 環境

Production（Cloudflare Pages ビルド環境）
- pnpm: 10.4.1
- Node.js: 22.16.0
- wrangler: 4.63.0（npx 経由でインストール）

## 追加情報

### 対応方針

プロジェクトルートに `wrangler.jsonc` を作成し、静的アセットのデプロイ設定を追加する:

```jsonc
{
  "name": "my-portfolio",
  "compatibility_date": "2026-02-07",
  "assets": {
    "directory": "./dist"
  }
}
```

### 関連Issue

同デプロイログ内で検出された副次的な警告:
- [ ] #61 shadcn-ui パッケージの bin 作成警告
- [ ] #62 Storybook builder-vite プリセット読み込みエラー
- [ ] #63 pnpm によるビルドスクリプト無視の警告

--- a/wrangler.jsonc
+++ b/wrangler.jsonc
@@ -1,0 +1,7 @@
+{
+  "name": "my-portfolio",
+  "compatibility_date": "2026-02-07",
+  "assets": {
+    "directory": "./dist"
+  }
+}

